<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomActionsDialog</class>
 <widget class="QDialog" name="RoomActionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Performing Room Action</string>
  </property>
  <widget class="QTextEdit" name="roomNameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please Choose Your Desired Room Action</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>INVITE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KICK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LEAVE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="instructionLabel_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please Enter the Target Room Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please Enter the Target User/Client Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="clientNameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
